nautobot/core/views/generic.py

```diff
@@ -403,6 +408,10 @@ def post(self, request, *args, **kwargs):
         form = self.model_form(data=request.POST, files=request.FILES, instance=obj)
         restrict_form_fields(form, request.user)
 
+        relationships_errors = get_relationships_errors(request, obj)
+        if len(relationships_errors) > 0:
+            return redirect(get_route_for_model(obj, "list"))
```

Comment:
This also seems undesirable to me - if there are missing required relationships, we should remain on the "edit" page and tell the user exactly what's missing, just as in the case of any other form validation error, not send them back to the list view.

Can we instead make it so that `form.is_valid()` below includes the validation of relationships and reports them as form errors if any?